app/models/review.rb

```diff
@@ -1,7 +1,17 @@
 class Review < ActiveRecord::Base
   belongs_to :restaurant
   belongs_to :user
+  has_many :votes
```

Comment:
what happens to votes if the review is destroyed? I would recommend a destroy dependency here so votes are cleaned up after a review is destroyed